docs/DIRECTORY_STRUCTURE.md

```diff
@@ -1,24 +1,27 @@
-# Project Directory Structure
+# Directory Structure
 
 Welcome to our project! This document outlines our project's directory structure, adhering to a "Feature-Driven Folder Structure" approach. The structure is inspired by insights from both an [English article](https://dev.to/profydev/screaming-architecture-evolution-of-a-react-folder-structure-4g25#indexjs-as-public-api) and a [Japanese article](https://zenn.dev/necscat/articles/d5d9b7a3f859d7).
 
-## Directory Structure Overview
+## Overview
 
-- [Project Directory Structure](#project-directory-structure)
-  - [Directory Structure Overview](#directory-structure-overview)
+- [Directory Structure](#directory-structure)
+  - [Overview](#overview)
     - [`app/`](#app)
+    - [`assets/`](#assets)
+    - [`assets/images/`](#assetsimages)
+    - [`assets/images/icons/`](#assetsimagesicons)
+    - [`assets/images/logos/`](#assetsimageslogos)
```

Comment:
What about `ui` directory inside `assets/images`?

Reply:
Oh, I was referring to the older `develop` branch, which didn't have `assets/images/ui/`. I will add it!